<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>668</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>BDTcomparator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/Tango/scalable/apps/accessories-calculator.svg</normaloff>:/icons/Tango/scalable/apps/accessories-calculator.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="inputTab">
       <attribute name="title">
        <string>Input</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="otputTab">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="resultsComboBox">
          <property name="maxVisibleItems">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="resultsView">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>DejaVu Sans Mono</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="optionsBox">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="GSgroupBox">
         <property name="title">
          <string>Gold standard</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QComboBox" name="GScomboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="confidenceLevelGroupBox">
         <property name="title">
          <string>Confidence level for performance measures</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QDoubleSpinBox" name="confidenceLevelSpinBox">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>0.800000000000000</double>
            </property>
            <property name="maximum">
             <double>0.999900000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.950000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="highlightGroupBox">
         <property name="title">
          <string>Confidence level for pairwise comparisons</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QDoubleSpinBox" name="pcConfidenceLevelSpinBox">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>0.800000000000000</double>
            </property>
            <property name="maximum">
             <double>0.999900000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.950000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="infoBox">
         <property name="title">
          <string>Output info</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="infoLabel1">
            <property name="font">
             <font>
              <family>DejaVu Sans Mono</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="infoLabel2">
            <property name="font">
             <font>
              <family>DejaVu Sans Mono</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="infoLabel3">
            <property name="font">
             <font>
              <family>DejaVu Sans Mono</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionCalculate"/>
   <addaction name="separator"/>
   <addaction name="actionSave_Results"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Tango/scalable/actions/document-open.svg</normaloff>:/icons/Tango/scalable/actions/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Tango/scalable/actions/document-save.svg</normaloff>:/icons/Tango/scalable/actions/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Tango/scalable/actions/document-save-as.svg</normaloff>:/icons/Tango/scalable/actions/document-save-as.svg</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave_Results">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Tango/scalable/mimetypes/x-office-spreadsheet.svg</normaloff>:/icons/Tango/scalable/mimetypes/x-office-spreadsheet.svg</iconset>
   </property>
   <property name="text">
    <string>Save Results</string>
   </property>
  </action>
  <action name="actionCalculate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Tango/scalable/apps/accessories-calculator.svg</normaloff>:/icons/Tango/scalable/apps/accessories-calculator.svg</iconset>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Tango/scalable/apps/help-browser.svg</normaloff>:/icons/Tango/scalable/apps/help-browser.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
